### PR TITLE
RET-2927

### PR DESCRIPTION
**Pull request by ikirsanov**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RET-2927

### Change description ###
Add NotificationState to createSendNotification, fix tests

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java [hunk 1]
```diff
@@ -96,6 +97,11 @@ public void createSendNotification(CaseData caseData) {
         sendNotificationType.setSendNotificationFullName2(caseData.getSendNotificationFullName2());
         sendNotificationType.setSendNotificationDetails(caseData.getSendNotificationDetails());
         sendNotificationType.setSendNotificationRequestMadeBy(caseData.getSendNotificationRequestMadeBy());
+        if (RESPONDENT_ONLY.equals(sendNotificationType.getSendNotificationSelectParties())) {
+            sendNotificationType.setNotificationState(NOT_VIEWED_YET);
```
[the remark this answers is not in the document]
Comment (ikirsanov) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java:101]: https://www.figma.com/file/V5EurhULwh0I1gHDUai6lV/Provide-something-the-tribunal-has-asked-for?node-id=19889-39426  "All orders and requests" list page bold text on the right.

**Review by lvo700 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java [hunk 2]
```diff
@@ -33,13 +33,14 @@
...
 import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.Helper.createLinkForUploadedDocument;
+import static uk.gov.hmcts.ethos.replacement.docmosis.service.TseService.NOT_STARTED_YET;
 
 @Service("sendNotificationService")
 @RequiredArgsConstructor
 @Slf4j
 @SuppressWarnings({"PMD.ExcessiveImports"})
 public class SendNotificationService {
-
+    public static final String NOT_VIEWED_YET = "notViewedYet";
```
Comment (JackReevies) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java:43]: Is this more suitable for et-common?

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationServiceTest.java [hunk 3]
```diff
@@ -99,6 +99,7 @@ void testCreateSendNotification() {
         assertEquals("John Doe", sendNotificationType.getSendNotificationFullName2());
         assertEquals("details", sendNotificationType.getSendNotificationDetails());
         assertEquals("Judge", sendNotificationType.getSendNotificationRequestMadeBy());
+        assertEquals("notStartedYet", sendNotificationType.getNotificationState());
```
Comment (JackReevies) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationServiceTest.java:102]: You've added an if statement with 2 branches, but only 1 test for this. Is there a branch not covered?

**Review by JackReevies — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java (hunk 2 above)
Comment (ikirsanov) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java:43]: Not sure, another one I'm using is in callbacks NOT_STARTED_YET

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationServiceTest.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (ikirsanov) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationServiceTest.java:98]: Done

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java (hunk 2 above)
Comment (ikirsanov) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java]: DOne

**Review by lvo700 — APPROVED** [review]

**Review by paul-peasegood — APPROVED** [review]

**Review by JackReevies — APPROVED** [review]

**Pull request merged**